experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -435,6 +442,9 @@ export const FormAudience = ({
 
   const isDesktop =
     experiment.application === NimbusExperimentApplicationEnum.DESKTOP;
+  const isMobile =
+    experiment.application != null &&
```

Comment:
I think from the flow perspective we will always have application selected, then only audience page will render

Reply:
I know but you know, typescript....